### PR TITLE
Update release-note with GraphQL Changes

### PR DESCRIPTION
**Pull request by DimuthuMadushan**

### CODE REVIEW

**Review by ThisaruGuruge — APPROVED** [review]

File: release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md [hunk 1]
```diff
@@ -58,10 +58,19 @@ To view bug fixes, see the [GitHub milestone for 2201.0.0 (Swan Lake)](https://g
...
+- Added support for disabling introspection queries
+- Added support for GraphQL interfaces
+- Added support for interfaces implementing interfaces
+- Introduced GraphQL client configurations
+
 ### Improvements
 
 #### `graphql` package
 
+- Added service-level interceptor execution for records fields, maps & tables
```
Comment (praneesha) [inline release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md:70]:
```suggestion
- Added service-level interceptor execution for records fields, maps, and tables
```

File: release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md [hunk 2]
```diff
@@ -58,10 +58,19 @@ To view bug fixes, see the [GitHub milestone for 2201.0.0 (Swan Lake)](https://g
...
+- Added service-level interceptor execution for subscriptions
+- Added returning all the errors related to a GraphQL document in a single response
```
Comment (praneesha) [inline release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md:72]:
```suggestion
- Enabled returning all the errors related to a GraphQL document in a single response
```

Comment (ThisaruGuruge) [reply release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md]: `Enabled` does not convey the expression here. This is a change to the existing behavior, which returned responses separately for different validation steps. `Enabled` sounds like this is something we have `disabled` previously. Which isn't the case.

Comment (ThisaruGuruge) [reply release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md]:
```suggestion
- Changed to return all the errors related to a GraphQL document in a single response
```

Might be a better option.

Comment (praneesha) [reply release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md]: Shall we say  `Added the ability to return all the errors related to a GraphQL document in a single response`?

Comment (ThisaruGuruge) [reply release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md]: Still not perfect. This is the behavior now. There's no option to the user to change this behavior.

Comment (praneesha) [reply release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md]: +1 for this: `Changed to return all the errors related to a GraphQL document in a single response`

**Review by ThisaruGuruge — APPROVED** [review]

**Review by praneesha — APPROVED** [review]

**Pull request merged**